src/Bitrix24/Bitrix24Provider.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace SocialiteProviders\Bitrix24;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Bitrix24Provider extends AbstractProvider
```

Comment:
I would prefer to stay consistent with the other providers and name this class **Provider**

Reply:
Done